test/integration/env-config/env.json

```diff
@@ -0,0 +1,31 @@
+{
+  "NOTION_KEY": {
+    "description": "Notion API key",
+    "required": true
+  },
+  "DATABASE_SECRET": {
+    "description": "Postgres database password",
+    "required": true,
+    "env": {
+      "development": {
+        "value": "abc"
```

Comment:
I think this should be `defaultValue`, it should be overwriteable by having it in `.env` / `process.env`

Reply:
Updated name from `value` -> `defaultValue`